openfunction-context/types.go

```diff
@@ -3,7 +3,7 @@ package openfunctioncontext
 const (
 	GRPC        Protocol     = "gRPC"
 	HTTP        Protocol     = "HTTP"
-	Dapr        Runtime      = "Dapr"
+	Dapr        Runtime      = "OpenFuncAsync"
```

Comment:
the const is still called Dapr ?

Reply:
This is an emergency patch. A new PR will be created later to fix all issues during interfacing with OpenFunction Serving.

Reply:
ok